test/node-rpc-test.js

```diff
@@ -177,6 +177,13 @@ describe('RPC', function() {
       const connectionsCnt = await nclient.execute('getconnectioncount', []);
       assert.strictEqual(connectionsCnt, 1);
     });
+
+    it('should rpc setban a peer',async () => {
+      const result = await nclient.execute('setban',[`127.0.0.1:${ports.p2p}`,'add']);
```

Comment:
this is actually the incorrect port, and you are not actually banning the peer. `ports.p2p` is the port that our main node is listening on. `ports.p2p + 100` is the port the peer is listening on (see line 143) -- neither of these are the correct port to use however. You may need to get the peers actual hostname with:

```js
const info = await nclient.execute('getpeerinfo', []);
const banThisAddress = info[0].addr // will be something like 127.0.0.1:56823
```